src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,50 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T> {
```

Comment:
```suggestion
class Pair<T, K> {
```
Use different parameter for firstParameter and secondParameter; You should be able to use different types

Reply:
Fixed